src/pedal.cpp

```diff
@@ -58,13 +58,19 @@ int Pedal::GenerateMIDI(FunctorParams * functorParams)
     if (!HasDir()) return FUNCTOR_CONTINUE;
 
     int velocity;
+    int tickOffset = 0;
     switch (GetDir())
     {
     case pedalLog_DIR_down:
         velocity = 126;
```

Comment:
Why 126 instead of 127?  In any case any value in the range from 64 to 127 will work as sustain pedal on.